typescript/src/includes/calculator.html

```diff
@@ -10,11 +10,11 @@ <h4>Predict metal occupancies in vivo</h4>
   <thead>
     <tr>
       <td></td>
-      <th>Metal affinities of molecule (M)<br/>(dissociation constants)</th>
-      <th>∆G for complexation with the molecule (kJ mol-1)</th>
-      <th>Buffered metal concentration (M)</th>
-      <th>Intracellular available ∆G (kJ mol-1)</th>
-      <th>Molecule occupancy with metal</th>
+      <th class="tooltip">Metal Affinity<br>(M)<span class="tooltiptext">Metal affinities of molecule (M) (dissociation constants, K<sub>D</sub>)</span></th>
+      <th class="tooltip">∆G(kJ mol<sup>-1</sup>)<span class="tooltiptext">ΔG for complexation with the molecule<br>(kJ mol<sup>-1</sup>)</span></th>
+      <th class="tooltip">Metal Availability<br>(M)<span class="tooltiptext">Buffered available metal concentration (M)</span></th>
+      <th class="tooltip">Available ∆G<br>(kJ mol<sup>-1</sup>)<span class="tooltiptext">Buffered available free energy of metal<br>(kJ mol<sup>-1</sup>)</span></th>
+      <th class="tooltip">Occupancy<span class="tooltiptext">Molecule occupancy with metal under these availabilities</span></th>
```

Comment:
How do the cells look without the line breaks now? Wondering if it would be better to remove them and let them break naturally now they're shorter. We might need to add `&nbsp;` between `kJ` and `mol<sup>-1</sup>` to stop the breaks there.

Reply:
<img width="886" alt="Screenshot 2021-11-23 at 10 55 15" src="https://user-images.githubusercontent.com/73976500/143012071-69de25c0-8912-4442-998c-be6c9aeb4d10.png">
They all look fine except this one, should I add the `<br>` back just on this one?

Reply:
Looks good to me; can you add `white-space: nowrap;` to `<sub>` in the CSS to avoid the issue in the tooltip?